src/azure-cli/azure/cli/command_modules/security/_params.py

```diff
@@ -90,6 +90,30 @@
 # Adaptive Application Controls
 adaptive_application_controls_group_name = CLIArgumentType(option_list=('--group-name'), metave='GROUPNAME', help='Name of an application control VM/server group')
 
+# Automations
+automation_scopes_arg_type = CLIArgumentType(options_list=('--scopes'), metavar='SCOPES', help='A collection of scopes on which the security automations logic is applied')
```

Comment:
Unnecessary to add `option_list=('--scopes')` as CLI framework will automatically add `--scopes` to option_list for the argument `scopes` (only when you want a different argument name). Please update other arguments as well.